android/src/main/java/com/reactnativeasiapay/AsiapayModule.kt

```diff
@@ -132,7 +132,7 @@ class AsiapayModule(private val reactContext: ReactApplicationContext) : ReactCo
     nativePayData!!.setCurrCode(EnvBase.Currency.valueOf(currency))
     nativePayData!!.setPayType(if (payType == "N") EnvBase.PayType.NORMAL_PAYMENT else EnvBase.PayType.HOLD_PAYMENT)
     nativePayData!!.setLang(EnvBase.Language.ENGLISH)
-    nativePayData!!.googlePayAuth = EnvBase.GooglePayAuth.CRYPTOGRAM_3DS
```

Comment:
can you make it as a parameter so we can control it from JS code? 
same as ios part

Reply:
okay